packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -112,6 +112,7 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
     );
   }
 
+  @autobind
   private _renderItem(item: IBreadcrumbItem) {
```

Comment:
can you rename while you're here to `_onRenderItem`? For greater consistency. :)